org.ekstep.navigation-1.0/renderer/plugin.js

```diff
@@ -19,6 +19,7 @@
     org.ekstep.service.controller.loadNgModules(this._templatePath, this.controllerPath);
 
     EkstepRendererAPI.addEventListener("renderer:overlay:show", instance.showOrHideOverlay, instance);        
+    EkstepRendererAPI.addEventListener("renderer:stage:reset", instance.resetNavigation, instance);
```

Comment:
Add this code in the controller. It has below event listeners in the init() method 
EkstepRendererAPI.addEventListener("renderer:content:start", $scope.showCustomNavigation);
    EkstepRendererAPI.addEventListener("renderer:overlay:show", $scope.showCustomNavigation);
    EkstepRendererAPI.addEventListener("renderer:overlay:hide", $scope.hideCustomNavigation);
 
Add EkstepRendererAPI.addEventListener("renderer:stage:reset", instance.resetNavigation, instance); as well